### DISCUSSION TITLE
云桌面不联网情况下开发

### DISCUSSION THREAD

**Discussion by huangwentai** [alibaba/formily#1988] [Q&A 2.x] [answered]
可以支持配置支持在**云桌面不联网**情况下开发吗，表单大多用在内部系统

我看到在两个网址加载文件，能**配置一起打包**进来吗

https://unpkg.com

https://cdn.jsdelivr.net/

**Comment by janryWang** [comment]
unpkg？哪里有依赖

**Reply by huangwentai** [reply]
忘了说，是designable的
![image](https://user-images.githubusercontent.com/19923827/128993442-e3fb59c7-f059-48eb-abf6-96535a02f7b9.png)

**Comment by janryWang** [comment, marked as answer]
这是playground，是用户自己定制的，cdn.jsdelivr.net是monaco依赖的，这种改不了，要改的话，bundle体积太大了

**Reply by janryWang** [reply]
你真要改的话，建议自己fork代码

**Reply by huangwentai** [reply]
明白，非常感谢
